### PR TITLE
Fix patchless validate for 2.12.3

### PR DESCRIPTION
**Pull request by utopiabound**
2.12.3 patchless only runs on el7.7

Issue #1267

Signed-off-by: Nathaniel Clark <nclark@whamcloud.com>

### CODE REVIEW

File: base_managed_patchless.profile [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "validation": [
     {
       "description": "The profile is designed for version 7 of EL",
```
Comment (jgrund) [inline base_managed_patchless.profile]:
```suggestion
      "description": "The profile is designed for version 7.7 of EL",
```

File: tests/framework/integration/existing_filesystem_configuration/cluster_setup [hunk 2]
```diff
@@ -15,6 +15,9 @@ pdsh -l root -R ssh -S -w $(spacelist_to_commalist $CHROMA_MANAGER) "exec 2>&1;
 # Clean out any yum info, if this is a manual system not automated the underlying repos might have changed.
 yum clean all
 
+# Do upgrade because vagrant doesn't have el7.7 IML#1277
+$PROXY yum upgrade -y
```
Comment (johnsonw) [inline tests/framework/integration/existing_filesystem_configuration/cluster_setup]: Should we do:
```
yum -y upgrade --exclude=python2-iml*
```
?

File: tests/framework/integration/shared_storage_configuration/full_cluster/cluster_setup.in [hunk 3]
```diff
@@ -15,6 +15,9 @@ pdsh -l root -R ssh -S -w $(spacelist_to_commalist $CHROMA_MANAGER ${STORAGE_APP
 # Clean out any yum info, if this is a manual system not automated the underlying repos might have changed.
 yum clean all
 
+# Do upgrade because vagrant doesn't have el7.7 IML#1277
+$PROXY yum upgrade -y
```
Comment (johnsonw) [inline tests/framework/integration/shared_storage_configuration/full_cluster/cluster_setup.in]: Should we do:
```
yum -y upgrade --exclude=python2-iml*
```
?

Comment (jgrund) [reply tests/framework/integration/shared_storage_configuration/full_cluster/cluster_setup.in]: There shouldn't  be any IML at this point.

Comment (johnsonw) [reply tests/framework/integration/shared_storage_configuration/full_cluster/cluster_setup.in]: Ok i'm fine with this then.

**Pull request merged**